docs/spec/manifest-v2-2.md

```diff
@@ -0,0 +1,118 @@
+
+    Fields are:
+    
+    - **`mediaType`** *string*
+    
+        The MIME type of the referenced object.
+    
+    - **`length`** *int*
+    
+        The length in bytes of the object. This field exists so that a client
+        will have an expected size for the content before validating. If the
+        length of the retrieved content does not match the specified length,
+        the content should not be trusted.
+    
+    - **`digest`** *string*
+    
+        A string labeled with a digest algorithm prefix and the hex-encoded
+        value of that digest for the content.
+
+        Digest strings should be of the form:
+
+            <digest-alg-name>:<hex-encoded-digest>
+
+        The `<digest-alg-name>` should be alpha-numeric and may contain the
+        characters `+`, `-`, `_`, and `.`. Example `<digest-alg-name>`s are:
+
+            sha256
+            sha512
+            sha3_256
+            sha3_512
+
+        The `<hex-encoded-digest>` value must contain hexadecimal digits only
+        and should be long enough to encode the full digest produced by the
+        corresponding digest algorithm.
+
+- **`dependencies`** *array*
+
+    Dependencies are an array of JSON objects which describe any content which
+    the target depends on and can be found in the same repository. Those may be
+    generic blobs of data or other content manifests with their own
+    dependencies. The exact type and usage of the dependent objects is left to
+    the target object data to describe.
+
+    Fields are:
+    
+    - **`length`** *int*
+    
+        Same as the `length` field of the `target` object above.
+    
+    - **`digest`** *string*
+    
+        Same as the `digest` field of the `target` object above.
+
+    The ordering of these dependencies is not significant other than for
+    consistent content-addressability. If the target object relies on any
+    ordering for its dependencies it should enforce an ordering of its own
+    separate from this list.
+
+- **`labels`** *object*
+
+    Labels may be keyed to *any* JSON object, allowing content creators to
+    annotate their manifest with any additional metadata beyond those already
+    defined by this manifest format or contained within the target object. 
+
+### Example Manifest
+
+The following manifest has a digest of
+`sha256:becb000d438bd85b1b32e5192ca367f5cdff5235cb8f3949d4751b1fc7ba75c9`:
+
+```json
+{
+    "schemaVersion": 2,
+    "target": {
+        "mediaType": "application/vnd.docker.container.image.params.v1+json",
+        "length": 7023,
+        "digest": "sha256:b5b2b2c507a0944348e0303114d8d93aaaa081732b86451d9bce1f432a537bc7"
+    },
+    "dependencies": [
+        {
+            "length": 32654,
+            "digest": "sha256:e692418e4cbaf90ca69d05a66403747baa33ee08806650b51fab815ad7fc331f"
+        },
+        {
+            "length": 16724,
+            "digest": "sha256:3c3a4604a545cdc127456d94e421cd355bca5b528f4a9c1905b15da2eb4a4c6b"
```

Comment:
What happened to mediatype? That is a critical field so that one does not trust the remote entity for indication of what something is. This all breaks from the descriptors in distribution.